File: leetcode/easy/771/771.cpp
/*
 * Runtime: 0 ms, faster than 100.00%
 * Memory Usage: 6.4 MB, less than 64.77%
 **/
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        int cnt = 0;
        for(int i=0; S[i]; ++i) 
        {
            for(int j=0; j<J.size(); ++j) 
            {
                if(S[i] == J[j]) ++cnt;
            }
        }
        
        return cnt;
    }
};

File: leetcode/easy/771/771.js
/**
 * Runtime: 80 ms, faster than 79.03%
 * Memory Usage: 38.8 MB, less than 62.63%
 */
var numJewelsInStones = function(J, S) {
    let cnt = 0;
    S = S.split('');
    S.forEach(c => {
        (J.includes(c)) ? cnt += 1 : cnt;
    })
    
    return cnt;
};
